src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -254,6 +271,14 @@ private void styleFetchedListView(CheckListView<CitationRelationItem> listView)
                         vContainer.getChildren().addLast(openWeb);
                     }
 
+                    Button showEntrySource = IconTheme.JabRefIcons.SOURCE.asButton();
+                    showEntrySource.setTooltip(new Tooltip(Localization.lang("%0 source", databaseContext.getMode().getFormattedName())));
```

Comment:
I am not sure if it is really BibLaTeX in this case - the citatoin fetcher always uses the bibtex dialect? Can you check? If unsure, just always use "BibTeX".